src/test/common/fdc3.app-channels.ts

```diff
@@ -128,7 +128,7 @@ export function createAppChannelTests(cc: ChannelControl<any,any>, documentation
 
         const testChannel = await cc.createTestChannel("a-different-test-channel");
         await cc.setupAndValidateListener1(testChannel, "unexpected-context", errorMessage, () => { /*noop*/ })
-        await cc.openChannelApp(acTestId7, undefined, APP_CHANNEL_AND_BROADCAST)
+        await cc.openChannelApp(acTestId7, undefined, APP_CHANNEL_AND_BROADCAST_TWICE)
```

Comment:
I _think_ this change is making it broadcast instrument and contact contexts rather than just instrument. Doesn't really break the test at all, but varies slightly from the description above - but back towards the actual test definition 👍  - I can't suggest a correction to the description above but it should be fixed

Reply:
yes, this is based on the test specification